e2e/ibazel.go

```diff
@@ -157,11 +157,12 @@ func (i *IBazelTester) build(target string, additionalArgs []string) {
 	}
 }
 
-func (i *IBazelTester) run(target string, additionalArgs []string) {
+func (i *IBazelTester) run(target string, additionalArgs []string, bazelArgs []string) {
```

Comment:
Could you move the bazelArgs to be the 2nd parameter to match the order that they are passed into Bazel?

Reply:
Done :heavy_check_mark:. PTAL.